From 18f1d53b474ce60dbb943e08478e712e72ece02e Mon Sep 17 00:00:00 2001
From: Mike FABIAN <mfabian@redhat.com>
Date: Thu, 30 Apr 2015 16:44:03 +0200
Subject: [PATCH 2/5] Addition and fixes for translit_neutral
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- Change Ö U+00D6 LATIN CAPITAL LETTER O WITH STROKE → O (instead of → OE)
- Change ö U+00F6 LATIN SMALL LETTER O WITH STROKE → o (instead of → oe)
- Add ₹ U+20B9 INDIAN RUPEE SIGN → INR
- Add ₫ U+20AB DONG SIGN → Dong (in addition to "₫ → Đồng")
- Add many others from
  http://unicode.org/cldr/trac/browser/trunk/common/transforms/Latin-ASCII.xml
- Add some more currency signs suggested by Marko Myllynen
- Add another patch with more characters by Marko Myllynen
---
 localedata/locales/translit_neutral | 470 +++++++++++++++++++++++++++++++++++-
 1 file changed, 465 insertions(+), 5 deletions(-)

diff --git a/localedata/locales/translit_neutral b/localedata/locales/translit_neutral
index b5cc079..c16d6f0 100644
--- a/localedata/locales/translit_neutral
+++ b/localedata/locales/translit_neutral
@@ -18,30 +18,306 @@ include "translit_wide";""
 
 % NO-BREAK SPACE
 <U00A0> <U0020>
+% INVERTED EXCLAMATION MARK
+<U00A1> <U0021>
+% CENT SIGN
+<U00A2> <U0063>
+% POUND SIGN
+<U00A3> "<U0047><U0042><U0050>"
+% YEN SIGN
+<U00A5> "<U004A><U0050><U0059>"
+% BROKEN BAR
+<U00A6> <U007C>
 % COPYRIGHT SIGN
 <U00A9> "<U0028><U0043><U0029>"
 % LEFT-POINTING DOUBLE ANGLE QUOTATION MARK
 <U00AB> "<U003C><U003C>"
+% NOT SIGN
+<U00AC> <U0021>
 % SOFT HYPHEN
 <U00AD> <U002D>
 % REGISTERED SIGN
 <U00AE> "<U0028><U0052><U0029>"
+% PLUS-MINUS SIGN
+<U00B1> "<U002B><U002D>"
+% ACUTE ACCENT
+<U00B4> <U0027>
+% MIDDLE DOT
+<U00B7> <U002E>
 % CEDILLA
 <U00B8> <U002C>
 % RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK
 <U00BB> "<U003E><U003E>"
+% INVERTED QUESTION MARK
+<U00BF> <U003F>
+% LATIN CAPITAL LETTER ETH
+<U00D0> <U0044>
 % MULTIPLICATION SIGN
 <U00D7> <U0078>
 % LATIN CAPITAL LETTER AE
 <U00C6> "<U0041><U0045>"
 % LATIN CAPITAL LETTER O WITH STROKE
-<U00D8> "<U004F><U0045>"
+<U00D8> <U004F>
+% LATIN CAPITAL LETTER THORN
+<U00DE> "<U0054><U0048>"
 % LATIN SMALL LETTER SHARP S
 <U00DF> "<U0073><U0073>"
 % LATIN SMALL LETTER AE
 <U00E6> "<U0061><U0065>"
+% LATIN SMALL LETTER ETH
+<U00F0> <U0064>
+% DIVISION SIGN
+<U00F7> <U002F>
 % LATIN SMALL LETTER O WITH STROKE
-<U00F8> "<U006F><U0065>"
+<U00F8> <U006F>
+% LATIN SMALL LETTER THORN
+<U00FE> "<U0074><U0068>"
+% LATIN CAPITAL LETTER D WITH STROKE
+<U0110> <U0044>
+% LATIN SMALL LETTER D WITH STROKE
+<U0111> <U0064>
+% LATIN CAPITAL LETTER H WITH STROKE
+<U0126> <U0048>
+% LATIN SMALL LETTER H WITH STROKE
+<U0127> <U0068>
+% LATIN SMALL LETTER DOTLESS I
+<U0131> <U0069>
+% LATIN SMALL LETTER KRA (collates with q in DUCET)
+<U0138> <U0071>
+% LATIN CAPITAL LETTER L WITH MIDDLE DOT
+<U013F> <U004C>
+% LATIN SMALL LETTER L WITH MIDDLE DOT
+<U0140> <U006C>
+% LATIN CAPITAL LETTER L WITH STROKE
+<U0141> <U004C>
+% LATIN SMALL LETTER L WITH STROKE
+<U0142> <U006C>
+% LATIN CAPITAL LETTER ENG
+<U014A> <U004E>
+% LATIN SMALL LETTER ENG
+<U014B> <U006E>
+% LATIN SMALL LETTER N PRECEDED BY APOSTROPHE
+<U0149> "<U0027><U006E>"
+% LATIN CAPITAL LIGATURE OE
+<U0152> "<U004F><U0045>"
+% LATIN SMALL LIGATURE OE
+<U0153> "<U006F><U0065>"
+% LATIN CAPITAL LETTER T WITH STROKE
+<U0166> <U0054>
+% LATIN SMALL LETTER T WITH STROKE
+<U0167> <U0074>
+% LATIN SMALL LETTER B WITH STROKE
+<U0180> <U0062>
+% LATIN CAPITAL LETTER B WITH HOOK
+<U0181> <U0042>
+% LATIN CAPITAL LETTER B WITH TOPBAR
+<U0182> <U0042>
+% LATIN SMALL LETTER B WITH TOPBAR
+<U0183> <U0062>
+% LATIN CAPITAL LETTER C WITH HOOK
+<U0187> <U0043>
+% LATIN SMALL LETTER C WITH HOOK
+<U0188> <U0063>
+% LATIN CAPITAL LETTER AFRICAN D
+<U0189> <U0044>
+% LATIN CAPITAL LETTER D WITH HOOK
+<U018A> <U0044>
+% LATIN CAPITAL LETTER D WITH TOPBAR
+<U018B> <U0044>
+% LATIN SMALL LETTER D WITH TOPBAR
+<U018C> <U0064>
+% LATIN CAPITAL LETTER OPEN E
+<U0190> <U0045>
+% LATIN CAPITAL LETTER F WITH HOOK
+<U0191> <U0046>
+% LATIN SMALL LETTER F WITH HOOK
+<U0192> <U0066>
+% LATIN CAPITAL LETTER G WITH HOOK
+<U0193> <U0047>
+% LATIN SMALL LETTER HV
+<U0195> "<U0068><U0076>"
+% LATIN CAPITAL LETTER IOTA
+<U0196> <U0049>
+% LATIN CAPITAL LETTER I WITH STROKE
+<U0197> <U0049>
+% LATIN CAPITAL LETTER K WITH HOOK
+<U0198> <U004B>
+% LATIN SMALL LETTER K WITH HOOK
+<U0199> <U006B>
+% LATIN SMALL LETTER L WITH BAR
+<U019A> <U006C>
+% LATIN CAPITAL LETTER N WITH LEFT HOOK
+<U019D> <U004E>
+% LATIN SMALL LETTER N WITH LONG RIGHT LEG
+<U019E> <U006E>
+% LATIN CAPITAL LETTER OI
+<U01A2> "<U004F><U0049>"
+% LATIN SMALL LETTER OI
+<U01A3> "<U006F><U0069>"
+% LATIN CAPITAL LETTER P WITH HOOK
+<U01A4> <U0050>
+% LATIN SMALL LETTER P WITH HOOK
+<U01A5> <U0070>
+% LATIN SMALL LETTER T WITH PALATAL HOOK
+<U01AB> <U0074>
+% LATIN CAPITAL LETTER T WITH HOOK
+<U01AC> <U0054>
+% LATIN SMALL LETTER T WITH HOOK
+<U01AD> <U0074>
+% LATIN CAPITAL LETTER T WITH RETROFLEX HOOK
+<U01AE> <U0054>
+% LATIN CAPITAL LETTER V WITH HOOK
+<U01B2> <U0056>
+% LATIN CAPITAL LETTER Y WITH HOOK
+<U01B3> <U0059>
+% LATIN SMALL LETTER Y WITH HOOK
+<U01B4> <U0079>
+% LATIN CAPITAL LETTER Z WITH STROKE
+<U01B5> <U005A>
+% LATIN SMALL LETTER Z WITH STROKE
+<U01B6> <U007a>
+% LATIN CAPITAL LETTER G WITH STROKE
+<U01E4> <U0047>
+% LATIN SMALL LETTER G WITH STROKE
+<U01E5> <U0067>
+% LATIN SMALL LETTER D WITH CURL
+<U0221> <U0064>
+% LATIN CAPITAL LETTER Z WITH HOOK
+<U0224> <U005A>
+% LATIN SMALL LETTER Z WITH HOOK
+<U0225> <U007A>
+% LATIN SMALL LETTER L WITH CURL
+<U0234> <U006C>
+% LATIN SMALL LETTER N WITH CURL
+<U0235> <U006E>
+% LATIN SMALL LETTER T WITH CURL
+<U0236> <U0074>
+% LATIN SMALL LETTER DOTLESS J
+<U0237> <U006A>
+% LATIN SMALL LETTER DB DIGRAPH
+<U0238> "<U0064><U0062>"
+% LATIN SMALL LETTER QP DIGRAPH
+<U0239> "<U0071><U0070>"
+% LATIN CAPITAL LETTER A WITH STROKE
+<U023A> <U0041>
+% LATIN CAPITAL LETTER C WITH STROKE
+<U023B> <U0043>
+% LATIN SMALL LETTER C WITH STROKE
+<U023C> <U0063>
+% LATIN CAPITAL LETTER L WITH BAR
+<U023D> <U004C>
+% LATIN CAPITAL LETTER T WITH DIAGONAL STROKE
+<U023E> <U0054>
+% LATIN SMALL LETTER S WITH SWASH TAIL
+<U023F> <U0073>
+% LATIN SMALL LETTER Z WITH SWASH TAIL
+<U0240> <U007A>
+% LATIN CAPITAL LETTER B WITH STROKE
+<U0243> <U0042>
+% LATIN CAPITAL LETTER U BAR
+<U0244> <U0055>
+% LATIN CAPITAL LETTER E WITH STROKE
+<U0246> <U0045>
+% LATIN SMALL LETTER E WITH STROKE
+<U0247> <U0065>
+% LATIN CAPITAL LETTER J WITH STROKE
+<U0248> <U004A>
+% LATIN SMALL LETTER J WITH STROKE
+<U0249> <U006A>
+% LATIN CAPITAL LETTER R WITH STROKE
+<U024C> <U0052>
+% LATIN SMALL LETTER R WITH STROKE
+<U024D> <U0072>
+% LATIN CAPITAL LETTER Y WITH STROKE
+<U024E> <U0059>
+% LATIN SMALL LETTER Y WITH STROKE
+<U024F> <U0079>
+% LATIN SMALL LETTER B WITH HOOK
+<U0253> <U0062>
+% LATIN SMALL LETTER C WITH CURL
+<U0255> <U0063>
+% LATIN SMALL LETTER D WITH TAIL
+<U0256> <U0064>
+% LATIN SMALL LETTER D WITH HOOK
+<U0257> <U0064>
+% LATIN SMALL LETTER OPEN E
+<U025B> <U0065>
+% LATIN SMALL LETTER DOTLESS J WITH STROKE
+<U025F> <U006A>
+% LATIN SMALL LETTER G WITH HOOK
+<U0260> <U0067>
+% LATIN SMALL LETTER SCRIPT G
+<U0261> <U0067>
+% LATIN LETTER SMALL CAPITAL G
+<U0262> <U0047>
+% LATIN SMALL LETTER H WITH HOOK
+<U0266> <U0068>
+% LATIN SMALL LETTER HENG WITH HOOK
+<U0267> <U0068>
+% LATIN SMALL LETTER I WITH STROKE
+<U0268> <U0069>
+% LATIN LETTER SMALL CAPITAL I
+<U026A> <U0049>
+% LATIN SMALL LETTER L WITH MIDDLE TILDE
+<U026B> <U006C>
+% LATIN SMALL LETTER L WITH BELT
+<U026C> <U006C>
+% LATIN SMALL LETTER L WITH RETROFLEX HOOK
+<U026D> <U006C>
+% LATIN SMALL LETTER M WITH HOOK
+<U0271> <U006D>
+% LATIN SMALL LETTER N WITH LEFT HOOK
+<U0272> <U006E>
+% LATIN SMALL LETTER N WITH RETROFLEX HOOK
+<U0273> <U006E>
+% LATIN LETTER SMALL CAPITAL N
+<U0274> <U004E>
+% LATIN LETTER SMALL CAPITAL OE
+<U0276> "<U004F><U0045>"
+% LATIN SMALL LETTER R WITH LONG LEG
+<U027C> <U0072>
+% LATIN SMALL LETTER R WITH TAIL
+<U027D> <U0072>
+% LATIN SMALL LETTER R WITH FISHHOOK
+<U027E> <U0072>
+% LATIN LETTER SMALL CAPITAL R
+<U0280> <U0052>
+% LATIN SMALL LETTER S WITH HOOK
+<U0282> <U0073>
+% LATIN SMALL LETTER T WITH RETROFLEX HOOK
+<U0288> <U0074>
+% LATIN SMALL LETTER U BAR
+<U0289> <U0075>
+% LATIN SMALL LETTER V WITH HOOK
+<U028B> <U0076>
+% LATIN LETTER SMALL CAPITAL Y
+<U028F> <U0059>
+% LATIN SMALL LETTER Z WITH RETROFLEX HOOK
+<U0290> <U007A>
+% LATIN SMALL LETTER Z WITH CURL
+<U0291> <U007A>
+% LATIN LETTER SMALL CAPITAL B
+<U0299> <U0042>
+% LATIN LETTER SMALL CAPITAL G WITH HOOK
+<U029B> <U0047>
+% LATIN LETTER SMALL CAPITAL H
+<U029C> <U0048>
+% LATIN SMALL LETTER J WITH CROSSED-TAIL
+<U029D> <U006A>
+% LATIN LETTER SMALL CAPITAL L
+<U029F> <U004C>
+% LATIN SMALL LETTER Q WITH HOOK
+<U02A0> <U0071>
+% LATIN SMALL LETTER DZ DIGRAPH
+<U02A3> "<U0064><U007A>"
+% LATIN SMALL LETTER DZ DIGRAPH WITH CURL
+<U02A5> "<U0064><U007A>"
+% LATIN SMALL LETTER TS DIGRAPH
+<U02A6> "<U0074><U0073>"
+% LATIN SMALL LETTER LS DIGRAPH
+<U02AA> "<U006C><U0073>"
+% LATIN SMALL LETTER LZ DIGRAPH
+<U02AB> "<U006C><U007A>"
 % MODIFIER LETTER PRIME
 <U02B9> <U2032>;<U00B4>
 % MODIFIER LETTER DOUBLE PRIME
@@ -68,6 +344,138 @@ include "translit_wide";""
 <U02D0> <U003A>
 % SMALL TILDE
 <U02DC> <U007E>
+% DOUBLE ACUTE ACCENT
+<U02DD> "<U0027><U0027>"
+% LATIN LETTER SMALL CAPITAL A
+<U1D00> <U0041>
+% LATIN LETTER SMALL CAPITAL AE
+<U1D01> "<U0041><U0045>"
+% LATIN LETTER SMALL CAPITAL BARRED B
+<U1D03> <U0042>
+% LATIN LETTER SMALL CAPITAL C
+<U1D04> <U0043>
+% LATIN LETTER SMALL CAPITAL D
+<U1D05> <U0044>
+% LATIN LETTER SMALL CAPITAL ETH
+<U1D06> <U0044>
+% LATIN LETTER SMALL CAPITAL E
+<U1D07> <U0045>
+% LATIN LETTER SMALL CAPITAL J
+<U1D0A> <U004A>
+% LATIN LETTER SMALL CAPITAL K
+<U1D0B> <U004B>
+% LATIN LETTER SMALL CAPITAL L WITH STROKE
+<U1D0C> <U004C>
+% LATIN LETTER SMALL CAPITAL M
+<U1D0D> <U004D>
+% LATIN LETTER SMALL CAPITAL O
+<U1D0F> <U004F>
+% LATIN LETTER SMALL CAPITAL P
+<U1D18> <U0050>
+% LATIN LETTER SMALL CAPITAL T
+<U1D1B> <U0054>
+% LATIN LETTER SMALL CAPITAL U
+<U1D1C> <U0055>
+% LATIN LETTER SMALL CAPITAL V
+<U1D20> <U0056>
+% LATIN LETTER SMALL CAPITAL W
+<U1D21> <U0057>
+% LATIN LETTER SMALL CAPITAL Z
+<U1D22> <U005A>
+% LATIN SMALL LETTER UE
+<U1D6B> "<U0075><U0065>"
+% LATIN SMALL LETTER B WITH MIDDLE TILDE
+<U1D6C> <U0062>
+% LATIN SMALL LETTER D WITH MIDDLE TILDE
+<U1D6D> <U0064>
+% LATIN SMALL LETTER F WITH MIDDLE TILDE
+<U1D6E> <U0066>
+% LATIN SMALL LETTER M WITH MIDDLE TILDE
+<U1D6F> <U006D>
+% LATIN SMALL LETTER N WITH MIDDLE TILDE
+<U1D70> <U006E>
+% LATIN SMALL LETTER P WITH MIDDLE TILDE
+<U1D71> <U0070>
+% LATIN SMALL LETTER R WITH MIDDLE TILDE
+<U1D72> <U0072>
+% LATIN SMALL LETTER R WITH FISHHOOK AND MIDDLE TILDE
+<U1D73> <U0072>
+% LATIN SMALL LETTER S WITH MIDDLE TILDE
+<U1D74> <U0073>
+% LATIN SMALL LETTER T WITH MIDDLE TILDE
+<U1D75> <U0074>
+% LATIN SMALL LETTER Z WITH MIDDLE TILDE
+<U1D76> <U007A>
+% LATIN SMALL LETTER TH WITH STRIKETHROUGH
+<U1D7A> "<U0074><U0068>"
+% LATIN SMALL CAPITAL LETTER I WITH STROKE
+<U1D7B> <U0049>
+% LATIN SMALL LETTER P WITH STROKE
+<U1D7D> <U0070>
+% LATIN SMALL CAPITAL LETTER U WITH STROKE
+<U1D7E> <U0055>
+% LATIN SMALL LETTER B WITH PALATAL HOOK
+<U1D80> <U0062>
+% LATIN SMALL LETTER D WITH PALATAL HOOK
+<U1D81> <U0064>
+% LATIN SMALL LETTER F WITH PALATAL HOOK
+<U1D82> <U0066>
+% LATIN SMALL LETTER G WITH PALATAL HOOK
+<U1D83> <U0067>
+% LATIN SMALL LETTER K WITH PALATAL HOOK
+<U1D84> <U006B>
+% LATIN SMALL LETTER L WITH PALATAL HOOK
+<U1D85> <U006C>
+% LATIN SMALL LETTER M WITH PALATAL HOOK
+<U1D86> <U006D>
+% LATIN SMALL LETTER N WITH PALATAL HOOK
+<U1D87> <U006E>
+% LATIN SMALL LETTER P WITH PALATAL HOOK
+<U1D88> <U0070>
+% LATIN SMALL LETTER R WITH PALATAL HOOK
+<U1D89> <U0072>
+% LATIN SMALL LETTER S WITH PALATAL HOOK
+<U1D8A> <U0073>
+% LATIN SMALL LETTER V WITH PALATAL HOOK
+<U1D8C> <U0076>
+% LATIN SMALL LETTER X WITH PALATAL HOOK
+<U1D8D> <U0078>
+% LATIN SMALL LETTER Z WITH PALATAL HOOK
+<U1D8E> <U007A>
+% LATIN SMALL LETTER A WITH RETROFLEX HOOK
+<U1D8F> <U0061>
+% LATIN SMALL LETTER D WITH HOOK AND TAIL
+<U1D91> <U0064>
+% LATIN SMALL LETTER E WITH RETROFLEX HOOK
+<U1D92> <U0065>
+% LATIN SMALL LETTER OPEN E WITH RETROFLEX HOOK
+<U1D93> <U0065>
+% LATIN SMALL LETTER I WITH RETROFLEX HOOK
+<U1D96> <U0069>
+% LATIN SMALL LETTER U WITH RETROFLEX HOOK
+<U1D99> <U0075>
+% LATIN SMALL LETTER A WITH RIGHT HALF RING
+<U1E9A> <U0061>
+% LATIN SMALL LETTER LONG S WITH DOT ABOVE
+<U1E9B> <U0073>
+% LATIN SMALL LETTER LONG S WITH DIAGONAL STROKE
+<U1E9C> <U0073>
+% LATIN SMALL LETTER LONG S WITH HIGH STROKE
+<U1E9D> <U0073>
+% LATIN CAPITAL LETTER SHARP S
+<U1E9E> "<U0053><U0053>"
+% LATIN CAPITAL LETTER MIDDLE-WELSH LL
+<U1EFA> "<U004C><U004C>"
+% LATIN SMALL LETTER MIDDLE-WELSH LL
+<U1EFB> "<U006C><U006C>"
+% LATIN CAPITAL LETTER MIDDLE-WELSH V
+<U1EFC> <U0056>
+% LATIN SMALL LETTER MIDDLE-WELSH V
+<U1EFD> <U0076>
+% LATIN CAPITAL LETTER Y WITH LOOP
+<U1EFE> <U0059>
+% LATIN SMALL LETTER Y WITH LOOP
+<U1EFF> <U0079>
 % ZERO WIDTH SPACE
 <U200B> ""
 % HYPHEN
@@ -120,6 +528,8 @@ include "translit_wide";""
 <U2039> <U003C>
 % SINGLE RIGHT-POINTING ANGLE QUOTATION MARK
 <U203A> <U003E>
+% TIRONIAN SIGN ET
+<U204A> <U0026>
 % FRACTION SLASH
 <U2044> <U002F>
 % WORD JOINER
@@ -130,12 +540,36 @@ include "translit_wide";""
 <U2062> ""
 % INVISIBLE SEPARATOR
 <U2063> ""
+% EURO-CURRENCY SIGN
+<U20A0> "<U0043><U0045>"
 % COLON SIGN
 <U20A1> "<U0043><U003D>"
+% CRUZEIRO SIGN
+<U20A2> "<U0043><U0072>"
+% FRENCH FRANC SIGN
+<U20A3> "<U0046><U0072><U002E>"
+% LIRA SIGN
+<U20A4> "<U004C><U002E>"
+% PESETA SIGN
+<U20A7> "<U0050><U0074><U0073>"
+% WON SIGN
+<U20A9> "<U004B><U0052><U0057>"
 % DONG SIGN
-<U20AB> "<U0110><U1ED3><U006E><U0067>"
+<U20AB> "<U0110><U1ED3><U006E><U0067>";"<U0044><U006F><U006E><U0067>"
 % EURO SIGN
 <U20AC> "<U0045><U0055><U0052>"
+% DRACHMA SIGN
+<U20AF> "<U0047><U0052><U0044>"
+% HRYVNIA SIGN
+<U20B4> "<U0433><U0440><U043D>";"<U0055><U0041><U0048>"
+% INDIAN RUPEE SIGN
+<U20B9> "<U0049><U004E><U0052>"
+% TURKISH LIRA SIGN
+<U20BA> "<U0054><U004C>"
+% RUBLE SIGN
+<U20BD> "<U0440><U0443><U0431>";"<U0052><U0055><U0042>"
+% PRESCRIPTION TAKE
+<U211E> "<U0052><U0078>"
 % TRADE MARK SIGN
 <U2122> "<U0028><U0054><U004D><U0029>"
 % OHM SIGN
@@ -150,10 +584,10 @@ include "translit_wide";""
 <U2194> "<U003C><U002D><U003E>"
 % LEFTWARDS DOUBLE ARROW
 <U21D0> "<U003C><U003D>"
-% LEFT RIGHT DOUBLE ARROW
-<U21D4> "<U003C><U003D><U003E>"
 % RIGHTWARDS DOUBLE ARROW
 <U21D2> "<U003D><U003E>"
+% LEFT RIGHT DOUBLE ARROW
+<U21D4> "<U003C><U003D><U003E>"
 % MINUS SIGN
 <U2212> <U2013>;<U002D>
 % DIVISION SLASH
@@ -166,6 +600,8 @@ include "translit_wide";""
 <U2219> <U2022>;<U00B7>
 % DIVIDES
 <U2223> <U007C>
+% PARALLEL TO
+<U2225> "<U007C><U007C>"
 % RATIO
 <U2236> <U003A>
 % TILDE OPERATOR
@@ -280,8 +716,32 @@ include "translit_wide";""
 <U2534> <U002B>
 % BOX DRAWINGS LIGHT VERTICAL AND HORIZONTAL
 <U253C> <U002B>
+% BOX DRAWINGS LIGHT DIAGONAL UPPER RIGHT TO LOWER LEFT
+<U2571> <U002F>
+% BOX DRAWINGS LIGHT DIAGONAL UPPER LEFT TO LOWER RIGHT
+<U2572> <U005C>
 % WHITE BULLET
 <U25E6> <U006F>
+% WHITE SMILING FACE
+<U263A> "<U003A><U0029>"
+% BLACK SMILING FACE
+<U263B> "<U003A><U0029>"
+% MATHEMATICAL RIGHT DOUBLE ANGLE BRACKET⟫
+<U27EB> "<U003E><U003E>"
+% MATHEMATICAL LEFT DOUBLE ANGLE BRACKET
+<U27EA> "<U003C><U003C>"
+% MATHEMATICAL LEFT FLATTENED PARENTHESIS
+<U27EE> <U0028>
+% MATHEMATICAL RIGHT FLATTENED PARENTHESIS
+<U27EF> <U0029>
+% MATHEMATICAL RISING DIAGONAL
+<U27CB> <U002F>
+% MATHEMATICAL FALLING DIAGONAL
+<U27CD> <U005C>
+% LEFT ANGLE BRACKET
+<U3008> <U003C>
+% RIGHT ANGLE BRACKET
+<U3009> <U003E>
 % KATAKANA-HIRAGANA DOUBLE HYPHEN
 <U30A0> <U003D>
 % VARIATION SELECTOR-1
-- 
2.4.3


